src/extension.ts

```diff
@@ -31,8 +31,8 @@ export function activate(context: vscode.ExtensionContext) {
 
 	dartSdkRoot = util.findDartSdk(<string>context.globalState.get(stateLastKnownSdkPathName));
 	if (dartSdkRoot == null) {
-		vscode.window.showErrorMessage("Dart Code: Could not find a Dart SDK to use. " +
-			"Please add it to your PATH or set it in the extensions settings and reload");
+		vscode.window.showErrorMessage("Could not find a Dart SDK to use. " +
+			"Please install one (www.dartlang.org) add configure the 'dart.sdkPath' setting.");
```

Comment:
If you use an installer that puts it on your PATH you shouldn't need to configure it so I think this is a bit misleading (the option is really a fallback in case the detection fails (or override to test others), which I think works fine now)